### ISSUE TITLE
two swear words without space isn't registered

### ISSUE THREAD

**Issue by Yehuda-Edelstein** [snguyenthanh/better_profanity#50]
```
better_profanity.contains_profanity('shitshitshit')
# False
```

**Comment by showierdata9978** [comment]
#49

**Comment by DamyanBG** [comment]
Hello,

I want to help here. Assign me the issue.

**Comment by showierdata9978** [comment]
> Hello,
> 
> I want to help here. Assign me the issue.

i fixed this in #49

**Comment by DamyanBG** [comment]
I just tried to receive True with contains_profanity function for string JPUSSY and received False. Maybe I installed older version or the fix is not merged?

**Comment by showierdata9978** [comment]
the fix is not merged

**Comment by showierdata9978** [comment]
this lib seems to be unmaintained

**Comment by DamyanBG** [comment]
I fixed the issue with braking a string into substrings, but I think we have to think for better library or to make new.

**Comment by Xynonners** [comment]
> this lib seems to be unmaintained

oof, are there any working profanity libraries at all?

**Comment by showierdata9978** [comment]
> > this lib seems to be unmaintained
> 
> oof, are there any working profanity libraries at all?

nope, I used to partially maintain a fork but my use for its gone

**Comment by Xynonners** [comment]
> > > this lib seems to be unmaintained
> > 
> > 
> > oof, are there any working profanity libraries at all?
> 
> nope, I used to partially maintain a fork but my use for its gone

sad
the only usable one I've seen is in NodeJS

**Comment by DamyanBG** [comment]
> > > > this lib seems to be unmaintained
> > > 
> > > 
> > > oof, are there any working profanity libraries at all?
> > 
> > 
> > nope, I used to partially maintain a fork but my use for its gone
> 
> sad the only usable one I've seen is in NodeJS

Give me some link to see it, we can easily rewrite it in Python.

**Comment by Xynonners** [comment]
> > > > > this lib seems to be unmaintained
> > > > 
> > > > 
> > > > oof, are there any working profanity libraries at all?
> > > 
> > > 
> > > nope, I used to partially maintain a fork but my use for its gone
> > 
> > 
> > sad the only usable one I've seen is in NodeJS
> 
> Give me some link to see it, we can easily rewrite it in Python.

https://github.com/jo3-l/obscenity
